packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -362,6 +362,26 @@ export class FluidDevtools implements IFluidDevtools {
 		return this._disposed;
 	}
 
+	/**
+	 * TODO: Need Verification
+	 * Add new data to the devtools logger.
+	 */
+	public addContainerData(
```

Comment:
Intestested if we should add something like overwrite or delete apart from add (since current implementation only allows "adding" new data).

Reply:
As mentioned in the other comment, I don't think this class is the right place for this functionality.